scripts/jonCli/getPackageNames.js

```diff
@@ -0,0 +1,7 @@
+module.exports = paths => {
```

Comment:
Could this use [`lerna list`](https://github.com/lerna/lerna/tree/master/commands/list#lernalist)?

Reply:
Yeh I think Lerna could simplify this